src/platform/tests/TestCHIPoBLEStackMgr.cpp

```diff
@@ -17,25 +17,13 @@
 
 #include "platform/internal/CHIPDeviceLayerInternal.h"
```

Comment:
It seems this didn't work. I think Linux/CHIPDevicePlatformConfig.h needs this stanza

```
#if CHIP_ENABLE_OPENTHREAD
#define CHIP_DEVICE_CONFIG_ENABLE_THREAD 1
#endif
```

and remove the change to `BUILD.gn`